lib/src/cli.rs

```diff
@@ -285,6 +292,46 @@ fn tar_export(opts: &ExportOpts) -> Result<()> {
     Ok(())
 }
 
+// Check if there are any files in a directory.
+fn is_only_directories(path: &Path, error_count: &mut i32) -> Result<bool> {
+    for entry in fs::read_dir(path)? {
+        let entry = entry?;
+        let path = entry.path();
+
+        let metadata = fs::metadata(&path)?;
+
+        if metadata.is_dir() {
+            is_only_directories(&path, error_count)?;
+        } else {
+            *error_count += 1;
+            if error_count < &mut 20 {
+                println!("File {:?} found under '/var' found.", path);
```

Comment:
Duplicate "found".  Also let's use `eprintln!()` since this is an error.

Maybe: `eprintln!("Found file in /var: {}, path")`
?

Reply:
the path includes /var. In the latest commit I just changed it to `eprintln!("Found file: {:?}, path")` does that work?